### ISSUE TITLE
pulseaudio-equalizer-gtk fails to load - ValueError: could not convert string to float

### ISSUE THREAD

**Issue by dotpointer** [pulseaudio-equalizer-ladspa/equalizer#36]
I tried this on Debian 9 and I have used kernelOfTruth:s pulseaudio-equalizer before I tried to run this. I followed the readme instructions.

The `pulseaudiio-equalizer enable` command runs fine, except that it never enables the equalizer.

If I run pulseaudio-equalizer-gtk then I get the following error:
```
Getting settings...
Getting settings...
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/pulseeq/equalizer.py", line 349, in do_startup
    self.window = Equalizer(application=self)
  File "/usr/local/lib/python3.6/dist-packages/pulseeq/equalizer.py", line 302, in __init__
    scale.set_value(float(ladspa_controls[x]))
ValueError: could not convert string to float: 
Getting settings...
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/pulseeq/equalizer.py", line 367, in do_activate
    self.window = Equalizer(application=self)
  File "/usr/local/lib/python3.6/dist-packages/pulseeq/equalizer.py", line 302, in __init__
    scale.set_value(float(ladspa_controls[x]))
ValueError: could not convert string to float: 
```

Maybe this has something to do with previous configuration files or such, but I renamed equalizerrc* and also presets directory. I also ensured that there were no binaries from the other version left.

**Comment by QHopp** [comment]
You had the right idea, this file is the problem. Just delete it and launch `pulseaudio-equalizer-gtk` again, it will create a brand new config file which works this time

**Comment by dotpointer** [comment]
But I renamed it, does it find it anyway?

**Comment by QHopp** [comment]
If you renamed it should create a new config file with the correct name. Is it correct ? Or you still have the problem ?

**Comment by QHopp** [comment]
If you still have problems, here's my `/.config/pulse/equalizerrc` file :
https://hastebin.com/beyukoyifo.css

**Comment by dotpointer** [comment]
Not sure why, but when I tested pulseaudio-equalizer and pulseaudio-equalizer-gtk now without installing it, then they ran well. Equalizer was enabled and the GUI appears and is usable.

Found that there are two locations for `equalizerrc`:

- `~/.config/pulse/equalizerrc` - `pulseaudio-equalizer enable` tells it is using this.
- `~/.pulse/equalizerrc` - I renamed this, but the equalizer does not read it.

Tested to remove the config, but it did not regenerate, did not test the GUI, only disable/enable.

So, I guess this works then.
